tests/utils.go

```diff
@@ -101,3 +102,29 @@ func RemoveBridgeFromNode(node, bridgename string) error {
 
 	return nil
 }
+
+func PodSpec(name string, resourceRequirements v1.ResourceList) *v1.Pod {
+	req := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name: name,
+					// TODO: can anyone propose an better image here?
+					Image: "docker.io/mmirecki/hello:latest",
```

Comment:
Can you please sugggest a nicer image here (one that will not finish right away)

Reply:
Use any image you want and add Command: []string{"sleep", "99999"} here

Reply:
you can use something like this

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: pod-example
spec:
  containers:
  - name: samplepod
    command: ["/bin/bash", "-c", "sleep INF"]
    image: centos
```

Reply:
updated